setup.py

```diff
@@ -23,6 +23,7 @@
 
 install_requirements = [
     'django>=1.11',
+    'sqlparse>=0.2.4'
```

Comment:
Hmm :thinking: do we really need to install this if we don't use it directly? Isn't it a dependency of Django and therefore not our problem?

Reply:
When running `RunSQL('SELECT abc')`, django needs sqlparse. However when running `RunSQL(('SELECT abc', ))` it does not. Not sure why... Checkout https://github.com/django/django/blob/master/django/db/migrations/operations/special.py#L117 and https://docs.djangoproject.com/en/dev/releases/2.2/#sqlparse-is-required-dependency if you are intressted.

Now I solved it by passing a list instead of a string.

Reply:
Hmm, weird. But good to know that Django 2.2 will add sqlparse as deps